app/mailers/invitation_mailer.rb

```diff
@@ -1,9 +1,9 @@
 # frozen_string_literal: true
 
 class InvitationMailer < ApplicationMailer
-  def account_invitation(account_invitation)
+  def account_invitation(account_invitation, current_user)
```

Comment:
```suggestion
  def account_invitation(account_invitation)
```

the user can be taken from `account_invitation`

Reply:
@andreybakanovsky what about the user_id column inside account_invitation model? Isn't it the user who sent the invite?

Reply:
Got it.